Intermittent failure with "broken pipe"

```
=== RUN   TestCopyToProcessStdinWithProcessExitError
--- FAIL: TestCopyToProcessStdinWithProcessExitError (0.00s)
	Error Trace:	copy_test.go:1174
	Error:		Not equal: "exit status 1" (expected)
			        != "write |1: broken pipe" (actual)
	Messages:	An error with value "exit status 1" is expected but got "write |1: broken pipe".
```

--- a/utils/copy_test.go
+++ b/utils/copy_test.go
@@ -1148,7 +1148,8 @@ func TestCopyToProcessStdinWithProcessExitError(t *testing.T) {
 	defer os.RemoveAll(testPath)
 
 	binaryContent := `#!/bin/sh
-echo "stdout string $@"
+read stuff
+echo "stdout string $stuff"
 exit 1
 `
 	fakeCmdPath := path.Join(testPath, "binary")

